testsuite/shared/src/main/java/org/jboss/as/test/integration/security/common/CoreUtils.java

```diff
@@ -178,7 +178,7 @@ public static void applyUpdates(final List<ModelNode> updates, final ModelContro
 
     public static void applyUpdate(ModelNode update, final ModelControllerClient client) throws Exception {
         ModelNode result = client.execute(new OperationBuilder(update).build());
-        if (LOGGER.isInfoEnabled()) {
+        if (LOGGER.isDebugEnabled()) {
```

Comment:
less logging by default === less IO, less logging, --> faster execution --> faster CI execution.
You can still enable it if you want it.